style.css

```diff
@@ -538,6 +537,104 @@ body {
   color: #fff5e1;
 }
 
+#popup-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 100;
+  padding: 2rem 1rem;
+  background-color: rgba(151, 148, 147, 0.9);
+}
+
+.popup {
+  background-color: #fff;
+  padding: 1.5rem;
+  max-width: 100%;
+}
+
+.popup-heading {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 3rem;
+}
+
+.popup-heading-text {
+  font-size: 32px;
+  font-family: "Roboto", serif;
+  color: #172b4d;
+}
+
+.popup-heading-btn {
+  background: none;
+  outline: none;
+  border: none;
+  cursor: pointer;
+}
+
+.popup-heading-btn i {
+  color: #979493;
+  font-size: 22px;
+}
+
+.popup-btn {
+  display: flex;
+  list-style: none;
+  gap: 0.5rem;
+  margin-top: 1.5rem;
+}
+
+.popup-btn-item {
+  padding: 1rem;
+  margin: 4rem 0 2rem 0;
+  font-family: "Poppins", serif;
+  font-size: 13px;
+  font-weight: 600;
+  color: #3c3a39;
+  background: none;
+  border: 1px solid #dfe1e6;
+}
+
+.popup-img-container {
+  width: 100%;
+}
+
+.popup-img {
+  width: 100%;
+}
+
+.popup-disc {
+  font-family: "Inter", serif;
+  font-size: 16px;
+  color: #344563;
+  margin: 2.1rem 0;
+}
+
+.popup-disc-btn {
```

Comment:
As the previous reviewer requested, stick to the Figma template, decrease the size of popup and align buttons like below.
<img width="743" alt="Screenshot 2023-02-11 at 11 02 08 PM" src="https://user-images.githubusercontent.com/25887253/218279094-e7571eac-d521-4348-985e-25d12d0e69c7.png">
<img width="1208" alt="Screenshot 2023-02-11 at 11 00 17 PM" src="https://user-images.githubusercontent.com/25887253/218279089-a5f2d174-8134-46d2-8a97-8dc2ed6db97a.png">